nservicebus/messaging/publish-subscribe/how-to-pub-sub.md

```diff
@@ -8,62 +8,47 @@ redirects:
 
 ## How to publish a message?
 
-```C#
-var message = new MyMessage
-              { 
-                 Prop1 = v1; 
-                 Prop2 = v2; 
-              };
-Bus.Publish(message);
-```
-
- OR instantiate and publish all at once:
-
-```C#
-Bus.Publish<IMyMessage>( m => { 
-                                m.Prop1 = v1; 
-                                m.Prop2 = v2; 
-                              });
-```
-
-## How to subscribe to a message?
 
-To manually subscribe and unsubscribe from a message:
+<!-- import InstancePublish -->
+
+If you are using interfaces to define your event contracts you need to set the message properties by passing in a lambda. NServiceBus will then generate a proxy and set those properties. 
+
+<!-- import InterfacePublish -->
+
+## Automatic subscriptions
+
+NServiceBus will automatically setup subscriptions for you. Messages matching the following criteria will be auto subscribed at startup by default.
+
+1. Defined as a event either using `IEvent` or by the `.DefiningEventsAs` convention.
+2. At least one [message handler](/nservicebus/handlers/) for the given message
```

Comment:
do we need to call out sagas here as well? or is that assumed using the word Handler?